dragonfly/implementations/__init__.py

```diff
@@ -28,3 +28,9 @@
 # keep these out of sequence, they inherit from elsewhere in dragonfly
 from .lockin_provider import *
 from .muxer_provider import *
+
+#from .sql_snapshot import *
```

Comment:
roach2interface inherits from r2daq, roach_daq_run_interface inherits from daq_run_interface; both of these will be satisfied by straight alphabetic ordering